### DISCUSSION TITLE
Worklows controling workflows

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17068] [Feature improvement]
May I purpose:

1. Extending system by allowing workflows control each other (for instance manual workflow calling another to mass edit currently filtered records etc.)
2. AND/OR allow workflows to be executed in list view like other systems does. No need to worry about harm to data as this can be controlled via conditions or privileges. Anyway this is business and responsibility of admins

When you start working on large datasets above became to make incredible sense.

**Comment by s2b-git** [comment]
Good point. Especially workflow triggered from List View.

**Reply by ArniSamano** [reply]
necessary if you digging in large datasets
